REFERENCE.md

```diff
@@ -637,13 +637,13 @@ The following parameters are available in the `systemd::dropin_file` defined typ
 
 Data type: `Systemd::Unit`
 
-the The target unit file to create, the value will be set to the `filename` parameter as well
+the The target unit file to create
 
 ##### <a name="filename"></a>`filename`
 
 Data type: `Systemd::Dropin`
 
-The target unit file to create
+The target unit file to create, the value will default to the name of the type
 
 Default value: `$name`
```

Comment:
I wonder if this isn't redundant. It already generates the default value so in my modules I tend to avoid describing the default value unless it really is special. Perhaps it would be more useful to describe that it's really the filename and that the full path is a combination of `$path`, $unit` and `$filename`.